docs/source/introduction.rst

```diff
@@ -44,6 +44,11 @@ Running this example from your terminal should simply show you a log message
 saying `Hello world!` but, what exactly is happening there?
 
 
+.. note:: The ``ns.shutdown()`` call is present because we don't want examples
+   to run indefinetely. For more information on that method, refer to the
+   :ref:`shutting_down` section.
```

Comment:
Maybe "is here" rather than "is present"? Sounds weird to me as it is right now, but maybe it is correct. If "is present" sounds better to you, just keep it that way. :blush: